arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,642 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-27-preview",
+    "title": "ContainerRegistryManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "Operation"
+        ],
+        "description": "Checks whether the container registry name is available.",
+        "operationId": "Operation_CheckNameAvailability",

```

Comment:
I would say change it as per Amar's recommendation, all azure apis should behave more or less in same way. With next CDN swagger changes we will try to change it to similar way.

---
In reply to: [83991770](https://github.com/Azure/azure-rest-api-specs/pull/549#discussion_r83991770) [](ancestors = 83991770)